src/uwtools/tests/test_cli.py

```diff
@@ -12,113 +12,114 @@
 
 from uwtools import cli
 from uwtools.tests.support import logged
+from uwtools.utils.file import FORMAT
 
 # Test functions
 
 
-def test_abort(capsys):
+def test__abort(capsys):
```

Comment:
Just want to confirm that there are two underscores here, the first for the function name definition and the second for the private member `_abort`

If this is correct, is this the standard? There is a similar example of this in PR UW-259

Reply:
Yes, that's correct and, yes, I'd recommend this as a standard: It's `test_`, and then whatever the function/method name is, including another underscore if that's how the function/method is named.